app/models/job_tracker.rb

```diff
@@ -44,15 +55,17 @@ def total?
     status[:total]&.positive?
   end
 
-  def percent
-    return nil unless total?
-
-    (100.0 * progress) / total
-  end
-
   private
 
   def number_with_delimiter(*args)
     ActiveSupport::NumberHelper.number_to_delimited(*args)
   end
+
+  def in_sidekiq_set?(set)
+    set.new.map(&:item).any? { |j| j['jid'] == provider_job_id }
```

Comment:
```suggestion
    set.new.find_job(provider_job_id)
```

ought to be equivalent, although sidekiq warns:

>  # This is a slow, inefficient operation.  Do not use under normal conditions.
> https://github.com/mperham/sidekiq/blob/main/lib/sidekiq/api.rb#L279-L286

(although I'm pretty sure that warning applies to this implementation too). I'm not sure if there's an even better approach?

Reply:
It seems to work ok with a few hundred jobs in the set. Maybe we add a guard clause to the method where if the job set is larger than some number (500?) it punts and just displays everything in the active queue? I dunno. I don't see another way of getting organization specific feedback about their jobs statuses.

Reply:
The only other thought I had was to group jobs by age as some indicator that something could be wrong if the job has been sitting around for more than a few days or a week.

Reply:
ok. I guess we can see what happens 🤷‍♂️